### ISSUE TITLE
Autocomplete: Unable to select remove icon of chips through keyboard

### ISSUE THREAD

**Issue by akshayaqburst** [primefaces/primereact#7730]
### Describe the bug

Autocomplete: Unable to select remove icon of chips through keyboard.
To remove the chip, user needs to deselect the options through keyboard.

### System Information

```Shell
System:
    OS: Linux 5.15 Ubuntu 20.04.6 LTS (Focal Fossa)
    CPU: (8) x64 11th Gen Intel(R) Core(TM) i7-1165G7 @ 2.80GHz
    Memory: 808.32 MB / 7.47 GB
    Container: Yes
    Shell: 5.0.17 - /bin/bash
  Binaries:
    Node: 20.5.0 - ~/.nvm/versions/node/v20.5.0/bin/node
    npm: 9.8.0 - ~/.nvm/versions/node/v20.5.0/bin/npm
    pnpm: 9.15.2 - ~/.nvm/versions/node/v20.5.0/bin/pnpm
  npmPackages:
    react: 18.3.1 => 18.3.1
```

### Steps to reproduce the behavior

1. Go to https://primereact.org/autocomplete/#multiple
2. Select one or two items. The selected items are shown as chips.
3. Try to do tab navigation to focus chips. Chips are not getting focused.

### Expected behavior

Remove icon of chips can be focused while tabbing through Autocomplete selected chips. User should be able to press enter key to dismiss chips.

**Comment by akshayaqburst** [comment]
I will work on this.
